src/components/Nav.jsx

```diff
@@ -5,17 +5,71 @@ import brigadeLogo from '../images/cfc_logo_2021.svg'
 import slackLogo from '../images/slack.svg'
 import githubLogo from '../images/github.svg'
 import meetup from '../images/meetup.svg'
+import styled from 'styled-components'
+
+const StyledNav = styled.nav`
+  height: 6rem;
+  padding: 1.5rem 6rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: white;
+`
+
+const StyledMenuIcon = styled.button`
```

Comment:
A few things:
Let's leave this at display none for this PR. We'll add it back in for the mobile designs. 
Let's also give this a more specific name (I'd go with Hamburger Menu).